velox/expression/tests/CMakeLists.txt

```diff
@@ -17,40 +17,41 @@ add_subdirectory(utils)
 add_executable(
   velox_expression_test
   ArgumentTypeFuzzerTest.cpp
+  ArrayViewTest.cpp
```

Comment:
Thank you for fixing the order of the files here. Would you extract this change into a separate PR?

Reply:
Alright. I will create a new PR, and move `FunctionSignatureTest` to `FunctionSignatureBuilderTest `, thanks.